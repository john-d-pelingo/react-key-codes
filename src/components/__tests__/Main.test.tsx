import { render } from '@testing-library/react'
import React from 'react'

import { Main } from '../Main'

describe('components - Main', () => {
  it('mounts', () => {
    const { getByText } = render(<Main />)

    const element = getByText(/press a key/i)

    expect(element).toMatchSnapshot()
  })
})
