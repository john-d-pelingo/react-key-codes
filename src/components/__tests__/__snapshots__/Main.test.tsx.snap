// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components - Main mounts 1`] = `
<span
  class="press-something"
>
  Press a key in your keyboard
</span>
`;
